en/docs/design/api-monetization/monetizing-an-api.md

```diff
@@ -446,64 +446,51 @@ Let's use the [wso2-am-stripe-plugin](https://github.com/wso2-extensions/wso2-am
-                CREATE TABLE IF NOT EXISTS AM_POLICY_SUBSCRIPTION (
-                            POLICY_ID INTEGER DEFAULT NEXTVAL('AM_MONETIZATION_PLATFORM_CUSTOMERS'),
-                    ID INTEGER NOT NULL AUTO_INCREMENT,
-                    SUBSCRIBER_ID INTEGER NOT NULL,
-                    TENANT_ID INTEGER NOT NULL,
-                    CUSTOMER_ID VARCHAR(256) NOT NULL,    
-                    PRIMARY KEY (ID),
-                    FOREIGN KEY (SUBSCRIBER_ID) REFERENCES AM_SUBSCRIBER(SUBSCRIBER_ID) ON DELETE CASCADE
-                );
-            
-                CREATE SEQUENCE AM_MONETIZATION_SHARED_CUSTOMERS START WITH 1 INCREMENT BY 1;
-                CREATE TABLE IF NOT EXISTS AM_POLICY_SUBSCRIPTION (
-                            POLICY_ID INTEGER DEFAULT NEXTVAL('AM_MONETIZATION_SHARED_CUSTOMERS'),
-                    ID INTEGER NOT NULL AUTO_INCREMENT,
-                    APPLICATION_ID INTEGER NOT NULL,
-                    API_PROVIDER VARCHAR(256) NOT NULL,
-                    TENANT_ID INTEGER NOT NULL,
-                    SHARED_CUSTOMER_ID VARCHAR(256) NOT NULL,
-                    PARENT_CUSTOMER_ID INTEGER NOT NULL,    
-                    PRIMARY KEY (ID),
-                    FOREIGN KEY (APPLICATION_ID) REFERENCES AM_APPLICATION(APPLICATION_ID) ON DELETE CASCADE,
-                    FOREIGN KEY (PARENT_CUSTOMER_ID) REFERENCES AM_MONETIZATION_PLATFORM_CUSTOMERS(ID) ON DELETE CASCADE
-                );
-            
-                CREATE SEQUENCE AM_MONETIZATION_SUBSCRIPTIONS START WITH 1 INCREMENT BY 1;
-                CREATE TABLE IF NOT EXISTS AM_POLICY_SUBSCRIPTION (
-                            POLICY_ID INTEGER DEFAULT NEXTVAL('AM_MONETIZATION_SUBSCRIPTIONS'),
-                    ID INTEGER NOT NULL AUTO_INCREMENT,
-                    SUBSCRIBED_APPLICATION_ID INTEGER NOT NULL,
-                    SUBSCRIBED_API_ID INTEGER NOT NULL,
-                    TENANT_ID INTEGER NOT NULL,
-                    SUBSCRIPTION_ID VARCHAR(256) NOT NULL,
-                    SHARED_CUSTOMER_ID INTEGER NOT NULL,    
-                    PRIMARY KEY (ID),
-                    FOREIGN KEY (SUBSCRIBED_APPLICATION_ID) REFERENCES AM_APPLICATION(APPLICATION_ID) ON DELETE CASCADE,
-                    FOREIGN KEY (SUBSCRIBED_API_ID) REFERENCES AM_API(API_ID) ON DELETE CASCADE,
-                    FOREIGN KEY (SHARED_CUSTOMER_ID) REFERENCES AM_MONETIZATION_SHARED_CUSTOMERS(ID) ON DELETE CASCADE
-                );
+            CREATE TABLE IF NOT EXISTS AM_MONETIZATION (
+                API_ID INTEGER NOT NULL,
+                TIER_NAME VARCHAR(512),
+                STRIPE_PRODUCT_ID VARCHAR(512),
+                STRIPE_PLAN_ID VARCHAR(512),
+                CONSTRAINT fk_api FOREIGN KEY (API_ID) REFERENCES AM_API (API_ID) ON DELETE CASCADE
+            );
+
+            CREATE TABLE IF NOT EXISTS AM_POLICY_PLAN_MAPPING (
+                POLICY_UUID VARCHAR(256),
```

Comment:
Shall we double check the column name `POLICY_UUID`? Both `POLICY_ID` and `POLICY_UUID` are there in the old version